medusa/tv/series.py

```diff
@@ -1553,15 +1556,25 @@ def load_imdb_info(self):
             log.debug(u'{id}: IMDb returned invalid info for {imdb_id}, skipping update.',
                       {'id': self.indexerid, 'imdb_id': self.imdb_id})
             return
+            
+        tmdb = Tmdb()
+        tmdb_id = self.externals.get('tmdb_id')
+        country_code = ''
+        country = ''
+        if tmdb_id:
+            show_info = tmdb._get_show_by_id(tmdb_id)
+            country_code = show_info['series']['origin_country']
+            print(country_code)
+            country = babelfish.Country(country_code).name
 
         self.imdb_info = {
             'imdb_id': imdb_obj.imdb_id,
             'title': imdb_obj.title,
             'year': imdb_obj.year,
             'akas': '',
             'genres': '|'.join(imdb_obj.genres or ''),
-            'countries': '',
-            'country_codes': '',
+            'countries': country,
```

Comment:
I know it's the easiest solution now, but technically this is not imdb info. And it should not be stored as such, to prevent confusion in future. Just something to keep in mind.